### ISSUE TITLE
Error .vimrc.bundles in windows xp

### ISSUE THREAD

**Issue by bennybobw** [spf13/spf13-vim#213]
I'm trying to install on Windows XP and I'm getting the following error when I run spf13-vim-windows-xp-install.cmd

`C:\Documents and Settings\ben\My Documents\Downloads\spf13>call vim -u "
$BASE_DIR/.vimrc.bundles" - +BundleInstall! +BundleClean +qall
Error detected while processing C:\Documents and Settings\ben\.spf13-vim-3\.vimrc.bundles:
line   36: E107: Missing braces: vundle#rc()
line   43: E492: Not an editor command:         Bundle 'gmarik/vundle'
line   44: E492: Not an editor command:         Bundle 'MarcWeber/vim-addon-mw-utils'
line   45: E492: Not an editor command:         Bundle 'tomtom/tlib_vim'
line   67: E15: Invalid expression: ['general', 'neocomplcache', 'programming', 'php', 'ruby', 'python', 'javascript', 'html', 'misc', 'scala']
E15: Invalid expression: ['general', 'neocomplcache', 'programming', 'php', 'ruby', 'python', 'javascript', 'html', 'misc', 'scala']
line   76: E121: Undefined variable: g:spf13_bundle_groups
E116: Invalid arguments for function count(g:spf13_bundle_groups, 'general')`

**Comment by spf13** [comment]
Do you have git installed?  

## 

Steve Francia
@spf13

**Comment by bennybobw** [comment]
yeah, git --version gives me:
`git version 1.7.11.msysgit.1`

Another clue is that it's copying two directories "and" and "Settings" into the directory where I run the script from, as though it's not escaping the spaces correctly.

**Comment by spf13** [comment]
For some reason vundle didn't install properly. Usually it's because git isn't installed. Not sure why it didn't work. I'd try reinstalling or manually installing vundle.   

## 

Steve Francia
@spf13

**Comment by bennybobw** [comment]
Hm. So it was choking on this line in the normal command prompt
`vim -u "$BASE_DIR/.vimrc.bundles" - +BundleInstall! +BundleClean +qall`

But when I ran it from git bash it worked (without $BASE_DIR). Does the whole thing have to be run from git bash? Maybe that was my issue. Or it could be that when I installed git I had the "Use git bash only" setting checked under the path settings.

Thanks for your help. Love your work on this. Hopefully this issue will help others if they run into a similar issue.
